### PR TITLE
Data collection ux improvements

### PR DESCRIPTION
**Pull request by emmabckstrm**
## Question creation, before & after
Screenshot 2021-09-29 at 15 17 19
Screenshot 2021-09-29 at 15 15 31

## Calculator overview, before & after
Screenshot 2021-09-29 at 15 17 09
Screenshot 2021-09-29 at 15 17 31

### CODE REVIEW

**Comment by emmabckstrm** [comment]
[Iterate on layout for editing questions–make units/question types clearer and mimic end result closer](https://trello.com/c/cUAaHn96/678-iterate-on-layout-for-editing-questions-make-units-question-types-clearer-and-mimic-end-result-closer)

**Comment by emmabckstrm** [comment]
[Make behavior when pressing enter while editing business calculator forms more intuitive](https://trello.com/c/iXfIuKGU/679-make-behavior-when-pressing-enter-while-editing-business-calculator-forms-more-intuitive)

**Comment by emmabckstrm** [comment]
[Don't open a new tab when previewing a business calculator](https://trello.com/c/Or5iAsgi/680-dont-open-a-new-tab-when-previewing-a-business-calculator)

**Comment by emmabckstrm** [comment]
[When reporting data, don't display the unit "Other" when there is only one unit](https://trello.com/c/Zk1AyR7O/682-when-reporting-data-dont-display-the-unit-other-when-there-is-only-one-unit)

**Comment by emmabckstrm** [comment]
[On organizations/show: show links to their reports](https://trello.com/c/NJGV8ZEu/671-on-organizations-show-show-links-to-their-reports)

**Comment by emmabckstrm** [comment]
Asking you to review, @stefanlindbohm , as it's basically only front end stuff :)

File: app/controllers/admin/organizations_controller.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def index
     end
 
     def show
+      @reports = ClimateReports::Report.where(organization_id: @organization.id)
```
Comment (stefanlindbohm) [inline app/controllers/admin/organizations_controller.rb:12]: This can be done directly with `@organization.reports`

**Review by stefanlindbohm — APPROVED** [review]
Nice improvements!

**Pull request merged**